content/realtime/channel-lifecycle.textile

```diff
@@ -0,0 +1,141 @@
+---
+title: Channel Metadata and Lifecycle Events
+section: realtime
+index: 15
+jump_to:
+  Help with:
+    - Overview#overview
+    - Metachannels#metachannels
+    - Lifecycle Events#lifecycle-events
+    - Reactor Rules#reactor
+    - Permissions#permissions
+  Channel API Types:
+    - ChannelDetails#channel-details
+    - ChannelStatus#channel-status
+    - Occupancy#occupancy
+---
+
+h2(#overview). Overview
+
+The realtime "channel lifecycle":#lifecycle-events and "occupancy":#occupancy events are published onto "metachannels":#metachannels that can be subscribed to using our existing "realtime libraries":/realtime. This is often used in conjunction with the "Ably Reactor":/reactor, which allows you to send this data to your "serverless endpoints":/general/functions, "queues":general/queues, and "streams":general/firehose.
```

Comment:
Perhaps:

Every channel in Ably's Data Stream Network that carries various messages has a "lifecycle":todo-link. The "channel lifecycle events":#lifecycle- are triggered whenever a channel changes its "lifecycle state":todo-link. A channel also consists of various occupants, meaning the participants that could either be publishers, subscribers, presence members or presence subscribers. The "occupancy":#occupancy events are triggered whenever there is a change in the number of any of these participants. Both the channel lifecycle and the occupancy events are published onto certain special channels called the "metachannels":#metachannels, that can be subscribed to, using our existing "realtime libraries":/realtime. This is often used in conjunction with the "Ably Reactor":/reactor, which allows you to send this data to your "serverless endpoints":/general/functions, "queues":general/queues, and "streams":general/firehose.